src/hooks/useAnalytics.ts

```diff
@@ -25,15 +25,17 @@ import { useApiState } from './useApiState';
 import { useBreakpoints } from './useBreakpoints';
 import { useDydxClient } from './useDydxClient';
 import { useSelectedNetwork } from './useSelectedNetwork';
-import { StatSigFlags, useStatSigGateValue } from './useStatsig';
+import { useAllStatsigGateValues } from './useStatsig';
 
 export const useAnalytics = () => {
   const latestTag = import.meta.env.VITE_LAST_TAG;
   const { walletType, walletConnectionType, evmAddress, dydxAddress, selectedWalletType } =
     useAccounts();
   const { indexerClient } = useDydxClient();
-  const isSkipEnabled = useStatSigGateValue(StatSigFlags.ffSkipMigration);
-
+  // all configs should be set by the time this provider loads
+  // we don't want to continuously ping statsig all the time to check for updates
+  // so we set it as a state item which holds referential equality
+  const [statsigConfig] = useState(useAllStatsigGateValues());
```

Comment:
I don't understand why this would be state instead of memo.

Reply:
useMemo depends on the dep array anyway so unless we did a deepcompare ourselves it would still reredner. keeping it in state maintains reference equality since we don't keep redefining it.

but you make a really good point below. by moving this to be a piece of state inside of the `useAllStatsigGateValues` hook, we can avoid needing to call it here :)

Reply:
oh wait i see, if we just useMemo with empty dep array, then it'd be fine. this makes sense